specutils/analysis/template_match.py

```diff
@@ -0,0 +1,36 @@
+from ..spectra.spectrum1d import Spectrum1D
+from ..spectra.spectrum_collection import SpectrumCollection
+from ..manipulation import FluxConservingResample
+from scipy.stats import chisquare
+
+def _template_match(observed_spectrum, template_spectrum):
```

Comment:
I would add a methods documentation section here to describe parameters and returns